testsuite/features/step_definitions/common_steps.rb

```diff
@@ -371,22 +371,71 @@
 
 # setup wizard
 
-When(/^I make the credentials primary$/) do
-  raise 'xpath: i.fa-star-o not found' unless find('i.fa-star-o').click
+Then(/^HTTP proxy verification should have succeeded$/) do
+  raise 'xpath: i.text-success not found' unless find('i.text-success', wait: DEFAULT_TIMEOUT)
 end
 
-When(/^I delete the primary credentials$/) do
-  raise 'xpath: i.fa-trash-o not found' unless find('i.fa-trash-o', match: :first).click
-  step 'I click on "Delete"'
+When(/^I enter the address of the HTTP proxy as "([^"]*)"/) do |hostname|
+  step %(I enter "#{$server_http_proxy}" as "#{hostname}")
+end
+
+When(/^I ask to add new credentials$/) do
+  raise 'xpath: i.fa-plus-circle not found' unless find('i.fa-plus-circle').click
+end
+
+When(/^I enter the SCC credentials$/) do
+  scc_username = ENV['SCC_CREDENTIALS'].split('|')[0]
+  scc_password = ENV['SCC_CREDENTIALS'].split('|')[1]
```

Comment:
I suggest:
scc_username, scc_password = ENV['SCC_CREDENTIALS'].split('|')

Reply:
does that work with arrays too?

Reply:
it does:

irb(main):002:0> a = ["tomato", "apple", "orange"]
=> ["tomato", "apple", "orange"]
irb(main):003:0> fruit1, fruit2, fruit3 = a
=> ["tomato", "apple", "orange"]
irb(main):004:0> puts fruit2
apple
=> nil